pkg/client/restclient/request.go

```diff
@@ -749,6 +749,16 @@ func (r *Request) Stream() (io.ReadCloser, error) {
 
 		result := r.transformResponse(resp, req)
 		if result.err != nil {
```

Comment:
Error() on result already does this correctly - so this entire block can be:

return result.Error()